marketplaceItems/vendingMachine/marketplaceResources/Description.md

```diff
@@ -1,3 +1,14 @@
-This vending machine is a blueprint item created by the Experiences team of High Fidelity.  This is an entity you can stage wherever you have edit rights that vends marketplace items.  A user selects what item to buy and clicking on it brings up the marketplace checkout page for that item.  If you check out the code for this item, you'll see we've made it really easy to repackage it with the marketplace items of your choice!  We can't wait to see what you come up with!
+This Vending Machine is a **Blueprint Item** created by the Experiences Team at High Fidelity.
 
-Code can be found [here]
+The Vending Machine is an entity you can stage wherever you have edit rights that vends marketplace items. After a user clicks on a Vending Machine's button, they will see the marketplace checkout page for that item.
+
+If you check out the code for this item, you'll see that we've made it really easy to repackage it with the marketplace items of your choice! We can't wait to see what you come up with. The Vending Machine code can be found [here](https://github.com/highfidelity/hifi-content/tree/master/marketplaceItems/vendingMachine).
```

Comment:
oh I actually had a problem doing this with the danceApp ```[here](https://github.com/highfidelity/hifi-content/tree/master/marketplaceItems/vendingMachine).```  It shows up right as a link you can go to on the marketplace app, but on the tablet app, you just see a here that you can't click.  Ahh reminds me I need to fix that.